src/main/webapp/i18n/en/course.json

```diff
@@ -78,6 +78,10 @@
                 "plural": "Test Courses"
             },
             "accessGroups": "Access Groups",
+            "learningPaths": {
+                "label": "Learning Paths Enabled",
+                "tooltip": "Enables generation of individualized learning paths for students consisting of competencies and their linked learning materials."
```

Comment:
feel free to ignore:
nitpick: IMO the English and the German sttrings are not equivalent because in German you talk about exercises only and in English you mention learning material, which is more holistic (e.g. containing lectures)

Reply:
Thank you for pointing this out. I changed the translation. Let me know if that works better 👍